catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -666,6 +667,11 @@ private Geometry getGeometry(String wkt) {
     private String sanitizeGeometry(String wkt) {
         try {
             Shape wktShape = WKT_READER.read(wkt);
+            //All polygons will be an instance of JtsGeometry. If it is not a polygon we don't need
```

Comment:
❓ Should we rename the method or add a comment on it to make its purpose clearer?